arch/arm/configs/bcm2709_defconfig

```diff
@@ -1382,8 +1382,10 @@ CONFIG_BCM2835_MBOX=y
 # CONFIG_IOMMU_SUPPORT is not set
 CONFIG_RASPBERRYPI_POWER=y
 CONFIG_IIO=m
+CONFIG_IIO_BUFFER=m
 CONFIG_IIO_BUFFER_CB=m
 CONFIG_IIO_SW_TRIGGER=m
+CONFIG_AS73211=m
```

Comment:
This is in the wrong place (well, a different place to where savedefconfig puts it).